docs/get_started/install.md

```diff
@@ -0,0 +1,347 @@
+      </div>
+    </div>
+  </div>
+</div>
+
+<!-- Linux Python GPU Options -->
+
+<div class="linux">
+  <div class="python">
+    <div class="gpu">
+      <div class="btn-group opt-group" role="group">
+        <button type="button" class="btn btn-default opt">Pip</button>
+        <button type="button" class="btn btn-default opt">Virtualenv</button>
+        <button type="button" class="btn btn-default opt">Docker</button>
+        <button type="button" class="btn btn-default opt">Build from Source</button>
+      </div>
+    </div>
+  </div>
+</div>
+
+<!-- END - Main Menu -->
+
+<!-- START - Linux Python CPU Installation Instructions -->
+
+<div class="linux">
+<div class="python">
+<div class="cpu">
+
+All the below installation instructions are tested on Ubuntu 14.04 and 16.04.
+
+<div class="virtualenv">
+<br/>
+
+**Step 1**  Install virtualenv for Ubuntu.
+
+```bash
+$ sudo apt-get install -y python-dev python-virtualenv
+```
+
+**Step 2**  Create and activate virtualenv environment for MXNet.
+
+Below command creates virtualenv environment at `~/mxnet` directory. However, you can choose any directory by replacing `~/mxnet` with directory of your choice.
+
+```bash
+$ virtualenv --system-site-packages ~/mxnet
+```
+
+Activate the virtualenv environment created for MXNet.
+
+```bash
+$ source ~/mxnet/bin/activate
+```
+
+After activating the environment, you should be seeing the prompt as below.
+
+```bash
+(mxnet)$
+```
+
+**Step 3**  Install MXNet in the active virtualenv environment.
+
+Installing MXNet with pip requires a latest version of `pip`. Install latest version of `pip` by issuing the below command in the terminal.
+
+```bash
+(mxnet)$ pip install --upgrade pip
+```
+
+Install MXNet with OpenBLAS acceleration.
+
+```bash
+(mxnet)$ pip install mxnet
+```
+
+**Step 4**  Validate the installation by running a simple MXNet code described [here](#validate-mxnet-installation).
+
+**Note**  You can read more about virtualenv [here](https://virtualenv.pypa.io/en/stable/userguide/).
+
+</div>
+
+<div class="pip">
+<br/>
+
+**Step 1**  Install prerequisites - wget and latest pip.
+
+Installing MXNet with pip requires a latest version of `pip`. Install latest version of `pip` by issuing the below command in the terminal.
+
+```bash
+$ sudo apt-get install wget
+$ wget https://bootstrap.pypa.io/get-pip.py && sudo python get-pip.py
+```
+
+**Step 2** Install MXNet with OpenBLAS acceleration.
+
+```bash
+$ pip install mxnet
+```
+
+**Step 3**  Validate the installation by running a simple MXNet code described [here](#validate-mxnet-installation).
+
+</div>
+
+<div class="docker">
+<br/>
+
+Docker images with MXNet are available at [Docker Hub](https://hub.docker.com/r/mxnet/).
+
+**Step 1**  Install Docker on your machine following the [docker installation instructions](https://docs.docker.com/engine/installation/linux/ubuntu/#install-using-the-repository).
```

Comment:
Install Docker on your machine *by* following the [docker installation instructions]